src/app/header.js

```diff
@@ -136,6 +137,7 @@ export class Header extends EventSubscriber {
              let url =
                 Misc.assembleImageLink(
                     this.context.server,
+                    this.context.getPrefixedURI(WEBCLIENT),
```

Comment:
Does this now have a trailing slash using ```reverse('webindex`)```, which is not handled?
assembleImageLink does:
```
let url = server + prefixed_uri + "/img_detail/" + image_id + '/?';
```